### ISSUE TITLE
prevent dropdown first item highlight after open

### ISSUE THREAD

**Issue by More4ever** [bootstrap-vue/bootstrap-vue#716]
So, is it posible to prevent highlight first dropdown item after open?

**Comment by tmorehouse** [comment]
For ARIA accessibility best practices, the first item in a dropdown should be focused when the dropdown is opened.  The current implementation emulated the functionality of a native select.  There is a change in the current master that ensures only one item has the highlight when the dropdown is opened.

We are currently re-working the focused state, where as previously the focus/hover state used the browser native outline (a blue box), which many people didn't like the look of, sowe went with making the drop down work more like native select element and native desktop menus.

**Comment by More4ever** [comment]
I think such behavior would be useful in dropdown-select component. But for default dropdown I think it would be useful and more flexibile to have ability not highlight first item

**Comment by tmorehouse** [comment]
Highlighting is part of the focusing (first item in dropdown is always focused for accessibility reasons).

Is it the look of the highlighting that you do not like?  We are addressing the "look" of the highlight, but will not remove the focusing of the first element (as is required by the W3 ARIA specifications for dropdown menus)

**Comment by More4ever** [comment]
I don't expect this behavior, as it doesn't exist in twitter bootstrap dropdown. I can rewrite styles for "active" class, but if component API will give me ability to chose, it would be great

**Comment by More4ever** [comment]
I can create pull request if it will be useful

**Comment by tmorehouse** [comment]
Bootstrap V4 will be moving to using keyboard navigation for Dropdowns (to be AIA compliant) in beta release and will exhibit this same behavior (it is in there current mater code from dropdown).

The main thing we will be changing is the look of the highlight/focus to be that of hover, which is more subtle than the current blue box.

**Comment by More4ever** [comment]
Ok, thanks a lot for your explanations

**Comment by tmorehouse** [comment]
PR #717 should address the styling of the focused element.  The first element is still focused automatically (as per ARIA specs), but is styled different  than the `active` state.

**Comment by zatziky** [comment]
Hi @tmorehouse ,

maybe I didn't understand quite well but Bootstrap doesn't have any item highligted upon opening a dropdown. Yet after hitting an arrow you can select the state. On item hover, the item item is highlighted but **without** the blue outline. Checkout the [docs](https://getbootstrap.com/docs/4.0/components/dropdowns/). 

The current behavior with the pre-selected blue outlined 1st item is really ugly and **not** the intended bootstrap way.

**After dropdown opens**

![image](https://user-images.githubusercontent.com/2794365/32267664-5267198e-beec-11e7-9541-f0eca5ef156f.png)

**On item hover**

![image](https://user-images.githubusercontent.com/2794365/32267702-776fd1f8-beec-11e7-98bd-4ea32e8d2521.png)


**On arrow keys**

![image](https://user-images.githubusercontent.com/2794365/32267735-8cc22006-beec-11e7-9732-eff53b7d12cd.png)

**Comment by mosinve** [comment]
![image](https://user-images.githubusercontent.com/20986566/32268927-f5ac6a08-bf01-11e7-8270-157baf1cf420.png)
This is current dropdown from BootstrapVue4.beta9 docs page
